include/net/azure_iot_hub.h

```diff
@@ -265,58 +371,29 @@ int azure_iot_hub_disconnect(void);
  *                     Azure IoT Hub.
  *
  *  @retval 0 If successful.
- *            Otherwise, a (negative) error code is returned.
- */
-int azure_iot_hub_send(const struct azure_iot_hub_data *const tx_data);
-
-/** @brief Receive incoming data from Azure IoT Hub.
- *
- *  @details Implicitly this function calls a non-blocking recv() on the
- *	     socket with IoT hub connection. It can be polled periodically,
- *	     but by default this functionality is handled internally in the
- *	     library.
- *
- *  @retval 0 If successful.
- *            Otherwise, a (negative) error code is returned.
+ *  @retval -EINVAL if a NULL pointer was provided.
+ *  @retval -ENOMSG if the provided message topic was invalid.
+ *  @retval -EMSGSIZE an internal buffer is too small to hold the topic data. This can for instance
+ *  @retval	      happen if message properties are in use, as they are appended to the topic.
```

Comment:
There shouldn't be a `@retval` on this line since it's a continuation of the previous line.